### PR TITLE
Allow to validate operations

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Access control: https://github.com/nextcloud/files_accesscontrol/pull/24
- [x] Automated tagging: https://github.com/nextcloud/files_automatedtagging/pull/10

@LukasReschke @MorrisJobke 

@karlitschek should backport to 10 before the final

### CODE REVIEW

**Comment by karlitschek** [comment]
please backport 👍

File: apps/systemtags/lib/Settings/Admin.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function getForm() {
 	 * @return string the section ID, e.g. 'sharing'
 	 */
 	public function getSection() {
-		return 'sharing';
+		return 'additional';
```
Comment (blizzz) [inline apps/systemtags/lib/Settings/Admin.php]: cc @jancborchardt you asked to put it into sharing

Comment (blizzz) [inline apps/systemtags/lib/Settings/Admin.php]: also seems a bit unrelated to the PR topic?

Comment (nickvergessen) [inline apps/systemtags/lib/Settings/Admin.php]: Well it took me 5 minutes to figure out where the tag management went to. This has nothing to do with sharing, when we have the workflow panel it should be there, until then it should be near the automated tagging panel, which is also in additional. Because that is what you use this thing with.

Comment (LukasReschke) [inline apps/systemtags/lib/Settings/Admin.php]: There's also an unit test for this, which makes it explode 💣 😉

Comment (nickvergessen) [inline apps/systemtags/lib/Settings/Admin.php]: Fixed

**Comment by MorrisJobke** [comment]
```
The autoloader is not up to date
Please run: bash build/autoloaderchecker.sh
And commit the result
```

**Comment by nickvergessen** [comment]
Updated the list

**Comment by MorrisJobke** [comment]
Tested and works 👍

**Comment by LukasReschke** [comment]
Needs a backport now :)

**Comment by nickvergessen** [comment]
Backport in https://github.com/nextcloud/server/pull/1001
